code/modules/mob/living/carbon/xenomorph/castes/Facehugger.dm

```diff
@@ -140,6 +140,12 @@
 	var/did_hug = hugger.attach(human, TRUE, 0.5)
 	if(client)
 		client?.player_data?.adjust_stat(PLAYER_STAT_FACEHUGS, STAT_CATEGORY_XENO, 1)
+	to_chat(src, SPAN_DEADSAY("<b>[human]</b> has been facehugged by <b>[src]</b> at \the <b>[loc.loc.name]</b>"))
+	for(var/mob/dead/observer/g in GLOB.observer_list)
+		to_chat(g, SPAN_DEADSAY("<b>[human]</b> has been facehugged by <b>[src]</b> at \the <b>[loc.loc.name]</b>" + " (<a href='?src=\ref[g];jumptocoord=1;X=[human.loc.x];Y=[human.loc.y];Z=[human.loc.z]'>JMP</a>)"))
```

Comment:
use a proper var name instead of g, also you can do `var/mob/dead/observer/ghost as anything in GLOB.observer_list` for added performance